specification/assets/gherkin/evaluation.feature

```diff
@@ -48,6 +48,19 @@ Feature: Flag evaluation
     Then the resolved object details value should be contain fields "showImages", "title", and "imagesPerPage", with values "true", "Check out these pics!" and 100, respectively
     And the variant should be "template", and the reason should be "STATIC"
 
+  Scenario: Passes evaluation details to after and finally hooks
```

Comment:
is this part of the evaluation, or should those new hooks be part of an "hooks.feature" annotated with `@hooks`? i think this is a little bit separated from the default evaluation and more an own thing in the spec